src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -410,10 +440,11 @@ class PackageDetailTab extends mixin(StoreMixin) {
           />
       },
       {
-        label: "Pre-Install Notes",
+        label: " ",
```

Comment:
Do we need this?

Reply:
Yes, with empty string it skips this and doesn't render the pre install notes.

Reply:
In the getItem (PackageDetailTab) we could probably change it to `if ((!label && label !== "") || !value) {` or something more explicit.
I'm just worried that we're introducing bad habits with this, as it a bit hacky, but I'm not blocking since it is a detail.